plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryConfig.java

```diff
@@ -224,6 +223,21 @@ public BigQueryConfig setCaseInsensitiveNameMatching(boolean caseInsensitiveName
         return this;
     }
 
+    @NotNull
+    @MinDuration("0ms")
+    public Duration getCaseInsensitiveNameMatchingCacheTtl()
+    {
+        return caseInsensitiveNameMatchingCacheTtl;
+    }
+
+    @Config("bigquery.case-insensitive-name-matching.cache-ttl")
+    @ConfigDescription("Duration for which case insensitive schema and table names are cached")
```

Comment:
Should we update descriptions here to tell users that setting it to 0ms disables the cache?